modules/developer_manual/examples/core/webdav_api/files_versions/successful-response-which-contains-files.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<d:multistatus xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns" xmlns:oc="http://owncloud.org/ns">
+  <d:response>
+    <d:href>/remote.php/dav/meta/43/v/</d:href>
+    <d:propstat>
+      <d:prop>
+        <d:resourcetype>
+          <d:collection/>
+        </d:resourcetype>
+      </d:prop>
+      <d:status>HTTP/1.1 200 OK</d:status>
+    </d:propstat>
+  </d:response>
+  <d:response>
+    <d:href>/remote.php/dav/meta/43/v/1576663297</d:href>
+    <d:propstat>
+      <d:prop>
+        <d:getlastmodified>Wed, 18 Dec 2019 10:01:37 GMT</d:getlastmodified>
+        <d:getcontentlength>163</d:getcontentlength>
+        <d:resourcetype/>
+        <d:getetag>"9cbeacc4d9ded5801341624599507048"</d:getetag>
+        <d:getcontenttype>text/plain</d:getcontenttype>
+      </d:prop>
+      <d:status>HTTP/1.1 200 OK</d:status>
+    </d:propstat>
+    <d:propstat>
+      <d:prop>
+        <d:quota-used-bytes/>
```

Comment:
I think you should simply not query these properties as they are not useful for versions

Reply:
They're not being specifically queried for. That said, I'll add a PROPFIND file to remove them.